samples/force-sensitive-resistor/FsrWithCapacitorSample.cs

```diff
@@ -8,12 +8,11 @@ namespace force_sensitive_resistor
 {
     class FsrWithCapacitorSample
     {
-        private GpioController _controller;
+        private GpioController _controller = new();
```

Comment:
Nifty :smile: I wasn't aware of this C#9 feature

Reply:
I know that this is just a sample but given you are touching this line we might as well just add the readonly to _controller 😆

Reply:
done